ds3/ds3.py

```diff
@@ -256,11 +256,17 @@ def __init__(self, ds3BulkResponse):
         self.bucketName = checkExistence(contents.bucket_name)
         if contents.cached_size_in_bytes:
             self.cachedSize = contents.cached_size_in_bytes
+        else:
+            self.cachedSize = 0
         if contents.completed_size_in_bytes:
             self.completedSize = contents.completed_size_in_bytes
+        else:
+            self.completedSize = 0
         self.jobId = checkExistence(contents.job_id)
         if contents.original_size_in_bytes:
             self.originalSize = contents.original_size_in_bytes
+        else:
+            self.originalSize = 0
```

Comment:
Should we change checkExistence to handle numbers so we don't have to write out if statements all the time? It could be given a second default parameter of what to return on failure, which would be None by default and set to 0 for numbers.